crates/uv/src/commands/tool/install.rs

```diff
@@ -178,9 +213,13 @@ pub(crate) async fn install(
         replace_symlink(&path, &target).context("Failed to install entrypoint")?;
         #[cfg(windows)]
         fs_err::copy(&path, &target).context("Failed to install entrypoint")?;
+        writeln!(printer.stderr(), "Installed `{name}`")?;
```

Comment:
Should we put this on one line if multiple? Like `Installed: black, blackd`

Reply:
I'm not sure. I kind of like that more here. I guess it depends if we'll include additional diagnostics about the installed entry points later.